tests/scripts/t3000/run_t3000_frequent_tests.sh

```diff
@@ -0,0 +1,121 @@
+
+  # Record the end time
+  end_time=$(date +%s)
+  duration=$((end_time - start_time))
+  echo "LOG_METAL: run_t3000_ethernet_tests $duration seconds to complete"
+}
+
+run_t3000_llama2_70b_tests() {
+  # Record the start time
+  start_time=$(date +%s)
+
+  echo "LOG_METAL: Running run_t3000_llama2_70b_tests"
+
+  # Llama2_70b related cached files and tests (the test should parse env variables similar to these)
+  export LLAMA_CKPT_DIR=/mnt/MLPerf/tt_dnn-models/llama-2/llama-2-70b-repacked/
+  export LLAMA_TOKENIZER_PATH=/mnt/MLPerf/tt_dnn-models/llama-2/tokenizer.model
+  export LLAMA_CACHE_PATH=/mnt/MLPerf/tt_dnn-models/llama-2/llama-data-cache/weights-cache-2
+
+  pytest models/demos/t3000/llama2_70b/tests/test_llama_mlp_t3000.py
+  pytest models/demos/t3000/llama2_70b/tests/test_llama_attention_t3000.py
+  pytest models/demos/t3000/llama2_70b/tests/test_llama_decoder_t3000.py
+  pytest models/demos/t3000/llama2_70b/tests/test_llama_model_t3000.py
+
+  # Record the end time
+  end_time=$(date +%s)
+  duration=$((end_time - start_time))
+  echo "LOG_METAL: run_t3000_llama2_70b_tests $duration seconds to complete"
+}
+
+run_t3000_mixtral_tests() {
+  # Record the start time
+  start_time=$(date +%s)
+
+  echo "LOG_METAL: Running run_t3000_mixtral_tests"
+
+  # Mistral8x7b 8 chip decode model test (env flags set inside the test)
+  pytest models/demos/t3000/mixtral8x7b/tests/test_mixtral_model.py::test_mixtral_model_inference[10-1-pcc]
+
+  # Record the end time
+  end_time=$(date +%s)
+  duration=$((end_time - start_time))
+  echo "LOG_METAL: run_t3000_mixtral_tests $duration seconds to complete"
+}
+
+run_t3000_tteager_tests() {
+  # Record the start time
+  start_time=$(date +%s)
+
+  echo "LOG_METAL: Running run_t3000_tteager_tests"
+
+  pytest tests/tt_eager/python_api_testing/unit_testing/misc/test_all_gather.py -k post_commit
+
+  # Record the end time
+  end_time=$(date +%s)
+  duration=$((end_time - start_time))
+  echo "LOG_METAL: run_t3000_tteager_tests $duration seconds to complete"
+}
+
+run_t3000_falcon40b_tests() {
+  # Record the start time
+  start_time=$(date +%s)
+
+  echo "LOG_METAL: Running run_t3000_falcon40b_tests"
+  
+  WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest models/demos/t3000/falcon40b/tests/test_falcon_causallm.py
+  WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest models/demos/t3000/falcon40b/tests/ci/test_falcon_end_to_end_1_layer_t3000.py
+
+  # Record the end time
+  end_time=$(date +%s)
+  duration=$((end_time - start_time))
+  echo "LOG_METAL: run_t3000_falcon40b_tests $duration seconds to complete"
+}
+
+run_t3000_tests() {
+  # Run ethernet tests
+  #run_t3000_ethernet_tests
+
+  # Run tteager tests
+  #run_t3000_tteager_tests
+
+  # Run llama2-70b tests
+  run_t3000_llama2_70b_tests
+
+  # Run mistral tests
+  run_t3000_mixtral_tests
+
+  # Run falcon40b tests
+  run_t3000_falcon40b_tests
+}
+
+main() {
+  if [[ -z "$TT_METAL_HOME" ]]; then
```

Comment:
Wonder if we can source this into a common script like @ttmchiou does for his docker script. So at the top of every test sh it would be

`source tests/scripts/test_common_checks.sh` or something like this

Reply:
will consolidate into the next refactor